### ISSUE TITLE
RsWithBody.java:38-41: Create a unit test for this class...

### ISSUE THREAD

**Issue by 0pdd** [artipie/http#37]
The puzzle `29-b88cdf36` from #29 has to be resolved: 

https://github.com/artipie/http/blob/c993c2b10014563b5bbde9115fe4b4b976e17f09/src/main/java/com/artipie/http/rs/RsWithBody.java#L38-L41

The puzzle was created by @g4s8 on 18-Feb-20. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by olenagerasimova** [comment]
@g4s8 assign me please

**Comment by 0crat** [comment]
The job [#37](https://github.com/artipie/http/issues/37) assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/654e1668-b297-4e3b-b1cb-bb69e9ae5bcc); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0pdd** [comment]
The puzzle `29-b88cdf36` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#144](https://github.com/artipie/http/issues/144) is still not solved.

**Comment by 0crat** [comment]
@olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by olenagerasimova** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/http/issues/37#issuecomment-627147511))

@olenagerasimova This is what I know about this job in CT2E6TK9B, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#37](https://github.com/artipie/http/issues/37) is [in scope](http://www.zerocracy.com/policy.html#14) for 5days
  * The role is `DEV`
  * The job is assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) for [5days](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @0pdd[/z](https://www.0crat.com/u/0pdd): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Fhttp%2337%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by olenagerasimova** [comment]
@g4s8 please, could you ask 0crat to close this issue?

**Comment by 0crat** [comment]
@olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) resigned from [#37](https://github.com/artipie/http/issues/37), since the job is not in scope anymore

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@47be01ee-43c3-4083-893b-28d1d64d4ea7, thread: PQ-CT2E6TK9B
com.zerocracy.claims.ClaimIn[214] java.lang.IllegalArgumentException: Parameter "age" not found in "Order was canceled" among: [reason, priority, job, cause, _parent_sig, flow, voluntarily, login]
```

[0.53.14](https://github.com/zerocracy/farm): CID: [8b5aa978-8b96-44dd-8a6d-09258f9f493a](https://www.0crat.com/footprint/CT2E6TK9B/8b5aa978-8b96-44dd-8a6d-09258f9f493a), Type: "Order was canceled"

**Comment by 0pdd** [comment]
@0pdd the puzzle [#165](https://github.com/artipie/http/issues/165) is still not solved; solved: [#144](https://github.com/artipie/http/issues/144).
